command/init.go

```diff
@@ -144,9 +144,70 @@ job "example" {
         }
       }
 
+      # The "artifact" stanza instructs Nomad to download an artifact from a
```

Comment:
Are there plans to do the rest? It will be a bit inconsistent if some have it and the rest don't